src/protagonist/Engine.Tests/Integration/ImageIngestTests.cs

```diff
@@ -122,6 +122,60 @@ public async Task IngestAsset_Success_HttpOrigin_AllOpen()
         storage.Size.Should().BeGreaterThan(0);
     }
     
+     [Fact]
+    public async Task IngestAsset_Success_OnLargerReingest()
+    {
+        // Arrange
+        // Create a new customer to have control over CustomerStorage and make sure it's isolated
+        const int customerId = -10;
+        var assetId = AssetId.FromString($"{customerId}/2/{nameof(IngestAsset_Success_OnLargerReingest)}");
+
+        // Note - API will have set this up before handing off
+        var origin = $"{apiStub.Address}/image";
+
+        var entity = await dbContext.Images.AddTestAsset(assetId, ingesting: true, origin: origin,
+            imageOptimisationPolicy: "fast-higher", mediaType: "image/tiff", width: 0, height: 0, duration: 0,
+            deliveryChannels: imageDeliveryChannels);
+        var asset = entity.Entity;
+        await dbContext.Customers.AddTestCustomer(customerId);
+        await dbContext.Spaces.AddTestSpace(customerId, 2);
+        dbContext.ImageStorages.Add(new ImageStorage()

```

Comment:
Worth adding an `AddTestImageStorage()` helper?

Reply:
The `imageStorage` passed in does change between the tests, so I'm not sure this is neccessary